app/src/main/java/org/traccar/client/TrackingController.java

```diff
@@ -100,7 +100,9 @@ public void onPositionUpdate(Position position) {
 
     @Override
     public void onNetworkUpdate(boolean isOnline) {
-        StatusActivity.addMessage(context.getString(R.string.status_connectivity_change));
+        String state_msg = context.getString((isOnline? R.string.status_connectivity_change_online : R.string.status_connectivity_change_offline ));
+
+        StatusActivity.addMessage(String.format("%s: %s", context.getString(R.string.status_connectivity_change), state_msg));
```

Comment:
I think it's probably better to just have 2 different full strings. Hardcoded formatting is not a very good idea.